examples/cluster/plot_kmedoids_digits.py

```diff
@@ -0,0 +1,78 @@
+"""
+=============================================================
+A demo of K-Medoids clustering on the handwritten digits data
+=============================================================
+
+In this example we compare different pairwise distance
+metrics for K-Medoids.
```

Comment:
It's not clear what the viewer is meant to get out of this example. I think it could be improved by limiting it to a couple of digits (avoiding the dense centre), and actually displaying those digits as in the data.

Either way it is standard to describe a bit more of exactly what is being plotted, and what we're meant to see from the output.

I'm also a bit concerned that PCA is biased towards Euclidean distances.